### DISCUSSION TITLE
Release note drafts

### DISCUSSION THREAD

**Discussion by forsyth2** [E3SM-Project/zppy#476] [General]
Since the markdown space for a release doesn't exist until the release is made, I'm using this space to draft release notes. Final release notes will be copied to https://github.com/E3SM-Project/zppy/releases.

Note: release notes do not display the title of a linked issue/pull request, but the markdown here does.

**Comment by xylar** [comment]
@forsyth2, you can draft a release.  
1. go to https://github.com/E3SM-Project/zppy/releases
2. Click "Draft a new release" at the top right
3. Type a draft
4. Hit "Save draft" at the bottom right
5. Do *not* hit "Publish release"
You can come back later and edit before you publish.

**Reply by forsyth2** [reply]
Oh wow, I completely missed that. I kept interpreting drafting a release as making a release candidate, not just a draft of the associated text.... Thanks!

**Comment by forsyth2** [comment]
v2.3.0
====

Incompatible API changes (user-facing)
- N/A

Features (user-facing)
- #402: Plugins for zppy
- #453: Allow generation of all variables
- #452: parallel option
- #459: Add hemispheric time series plots

Small Improvements (user-facing)
- #374: Fix input file error message
- #380: Print job ids to file
- #382: Refactor slurm header
- #389: Refactor coupled_global.py
- #381: Determine mapping file
- #417: Add support for clm2 cam in e3sm_to_cmip call
- #421: Support clm2 files
- #426: Allow more concise cfg
- #451: Help text
- #458: Update diags defaults
- #461: Make parallel optional for climo 

Bug Fixes (user-facing)
- #384: Fix Mache invocations
- #457: Automatically determine climo_diurnal input files
- #466: Fixes for rc4
- #472: Fix mvm paths

Documentation (user-facing)
- #375: Include Chrysalis cfgs
- #376: Include Perlmutter cfgs
- #377: Include Compy cfgs
- #378: Documentation on debugging
- #379: Improve git checkout in docs
- #433: Update release docs
- #436: Update defaults in docs
- #462: ncclimo vars documentation
- #460: Skip glb-only plots for hemispheric ts

Tests
- #447: Fixes for Perlmutter
- #448: Fixes for Compy
- #449: Testing fixes
- #464: Perlmutter fixes for rc3
- #463: Compy fixes for rc3

DevOps
- #395: Make setup.py consistent with conda recipe
- #410: Update release process
- #428: Switch docs to mamba and Mambaforge 
- #429: Change workflow to mamba
- #434: Fix GitHub Actions
- #435: Revert mamba
- #438: Change workflows to mamba, take 2

Version Updating
- #437: Bump to v2.3.0rc1 without tbump
- #443: Bump to v2.3.0rc2 without tbump 
- #465: Bump to 2.3.0rc3
- #469: Bump to 2.3.0rc4
- #473: Bump to 2.3.0rc5

Other code changes
- #432: Remove AMWG task
